docs/source/en/api/pipelines/text_to_video.md

```diff
@@ -167,6 +167,10 @@ Here are some sample outputs:
     </tr>
 </table>
 
+## Tips
+
+Video generation is memory-intensive and one way to reduce your memory usage is to set `enable_forward_chunking` on the pipeline's UNet so you don't run the entire feedforward layer at once. Breaking it up into chunks in a loop is more efficient.
```

Comment:
Should we mention the `text-img2vid` guide from the tip sections of SVD and text_to_video?